backend-api/app/models/producers.py

```diff
@@ -0,0 +1,21 @@
+from ..database import Base
+import sqlalchemy as sa
+from . import users
+from sqlalchemy.orm import relationship
+
+class Producer(users.User, Base):
+    __tablename__ = "Producers"
+    id = sa.Column(sa.Integer, sa.ForeignKey("Users.id"), primary_key=True, index=True, nullable=False)
+    eth_address = sa.Column(sa.String, unique=True,index=True,nullable=False)
+    name = sa.Column(sa.String, nullable=False)
+    email = sa.Column(sa.String, unique=True, index=True, nullable=False)
+    location_id = sa.Column(sa.Integer, sa.ForeignKey("Locations.id"), nullable=False)
```

Comment:
Let's using relational mapping instead

Reply:
By that, do you mean use the relation() function instead of making the location_id a foreign key? Because if so, that wouldn't work because when we define relationships using SQLAlchemy's relationship function, it is a high-level ORM 
 feature that helps facilitate querying and navigation between related objects in our Python code. SQLAlchemy's ORM allows you to work with related objects in a more object-oriented way, but it does not directly impact the actual database schema, which is why I decided to not use that call for one-to-many relations or many-to-one relations since a foreign key could just be used directly.

Reply:
You're right, my mistake 😅 . I was thinking this was like django.

Can we add location as a relationship like with ctegories underneath? That way, we can do `producer.location` 
https://docs.sqlalchemy.org/en/20/orm/basic_relationships.html

Reply:
We can definitely do that!